src/pages/components/footer.mdx

```diff
@@ -24,176 +26,215 @@ import ResourceLinks from 'components/ResourceLinks';
+
+The Footer is generated automatically based on the country and language code that is set on the page in the metadata.
 
 ![Image of default footer](../../images/component/footer/footerV2-default-01.jpg)
 
 <Caption>
   Default footer at 1584px max breakpoint, without optional adjunct legal links
 </Caption>
 
+## Default
+
+The Default footer includes all high-level corporate navigation, required legal links, locale selector, branding, and optional adjunct legal links.
+
 #### When to use
 
 Default footer is suitable for most of the IBM.com pages. Examples include the IBM.com US home page.
 
-## Variations
+### Anatomy
 
-### Short
-
-Short footer removes all the high-level corporate navigation, leaving the required legal links, the optional adjunct legal links, the locale selector and branding.
+![Image of minimal footer](../../images/component/footer/footerV2-default-anatomy.jpg)
 
-![Image of minimal footer](../../images/component/footer/footerV2-short-01.jpg)
+<Caption>
+  Default footer at the 1584px (max) breakpoint with one adjunct legal link
+</Caption>
 
-<Caption>Short footer at 1312px xlg breakpoint</Caption>
+1. **IBM logo:** Links to the IBM homepage and is a required branding element of the Footer. The IBM logo should not be altered.
+1. **Link group heading:** The link group heading is present for all the link groups.
+1. **Link:** The Default footer links are carefuly considered from the main navigation level and should not be changed.
+1. **Locale selector:** Use the locale selector to trigger the locale modal component to allow users to change geographic regions and languages, if available.
+1. **Legal links:** The general legal links are present on every IBM Footer and cannot be changed.
+1. **Adjunct legal link(s) (Optional)**: Use up to two adjunct legal links for notes or messages legally required to provide regarding changes, policies or trademark issues.
 
-#### When to use
+### Behavior
 
-Short footer is reserved for approved transactional pages, such as check out and registration experiences.
+Default footer spans the full width of the viewport and is the last element in the browser window. The Footer elements are persistent throughout the online experience. It is fully responsive and changes how elements are displayed based on the browser size.
 
-### Micro
+#### Max breakpoint
 
-Micro footer variation only shows the required legal links and the optional locale selector to maximize the space for the page content. Adjunct legal links are not allowed. If you need them, please use either the default, or the short footer.
+If the browser is wider than the max breakpoint of 1584px, the Default footer and all other page content will center and extra margin will appear on either side of the page layout.
 
-![Image of micro footer](../../images/component/footer/footerV2-micro-01.jpg)
+#### Md and sm breakpoints
```

Comment:
When referring to text that is used in code, should we use the exact case and wrap in code brackets, i.e. `md` and `sm`. Otherwise maybe we spell it out (Medium and small)? Just want to avoid any confusion when they are used.